services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaControllerITest.kt

```diff
@@ -303,10 +303,10 @@ class HankeKayttajaControllerITest(@Autowired override val mockMvc: MockMvc) : C
                 assertThat(id).isNotNull()
                 assertThat(etunimi).isEqualTo("test1")
                 assertThat(sukunimi).isEqualTo("name1")
-                assertThat(nimi).isEqualTo("test1 name1")
                 assertThat(puhelinnumero).isEqualTo("0405551111")
                 assertThat(sahkoposti).isEqualTo("email.1.address.com")
                 assertThat(tunnistautunut).isEqualTo(false)
+                assertThat(kutsuttu).isNotNull().isEqualTo(HankeKayttajaFactory.INVITATION_DATE)
```

Comment:
Is `isNotNull()` check needed or does `isEqualTo` cover also null case?

Reply:
Yes, `assertThat(null).isEqualTo(INVITATION_DATE)` fails, so the null check is not required. I removed it.